src/mnode/mnode-rpc.cpp

```diff
@@ -2424,6 +2473,27 @@ As json rpc
                 mnObj.pushKV("totalstoragefee", totalFee);
                 return mnObj;
             }
+            case RPC_CMD_LIST::validateusername: {
+                std::string username;
+                if (params.size() > 2) {
+                    username = params[2].get_str();
+
+                    UniValue obj(UniValue::VOBJ);
+                    std::string usernameValidationError;
+                    bool isBad = CChangeUsernameTicket::isUsernameBad(username, usernameValidationError);
+                    if (!isBad) {
+                        CChangeUsernameTicket existingTicket;
+                        if (CChangeUsernameTicket::FindTicketInDb(username, existingTicket)) {
+                            isBad = true;
+                            usernameValidationError = "Username is not valid, it is already registered";
+                        }
+                    }
+                    obj.pushKV("isBad", isBad);
+                    obj.pushKV("validationError", usernameValidationError);
```

Comment:
usernameValidationError is not used after that... just move it. obj.pushKV("validationError", move(usernameValidationError));

Reply:
fixed